flexget/api/core/authentication.py

```diff
@@ -18,42 +21,37 @@
 
 @login_manager.request_loader
 @with_session
-def load_user_from_request(request, session=None):
+def load_user_from_request(request: Request, session: Session = None) -> Optional[User]:
     auth_value = request.headers.get('Authorization')
 
     if not auth_value:
-        return
+        return None
```

Comment:
redundant usage of `None`

Reply:
If a function returns a value at one path, mypy expects:
- return values to be specified on all other returns and
- all possible paths to return a value

Simplified example:

```python
from typing import Optional

def foo(n: int) -> Optional[int]:
    if n > 10:
        return n
    elif n < 0:
        return
```
```
main.py:3: error: Missing return statement
main.py:7: error: Return value expected
Found 2 errors in 1 file (checked 1 source file)
```

Reply:
huh. makes sense typing wise I guess, but this kinda conflict with the pythonic approach. Not sure how I feel about this one. I guess if we want to run mypy then there's no other choice, right?